.github/workflows/on_dispatch_deploy_production.yml

```
@@ -0,0 +1,32 @@
+on: 
+  workflow_dispatch:
+    
+name: Deploy production static website
+
+jobs:
+  deploy-cms-production:
+    uses: ./.github/workflows/reusable_cms_deploy.yml
+    with:
+      environment: production 
+      workload_identity_provider_secret_name: passculture-metier-ehp/gcp_metier_prod_workload_identity_provider 
+      teleport_version: 13.4.4
+      teleport_proxy: teleport.passculture.team:443
+      teleport_kubernetes_cluster: passculture-metier-prod
+      image_tag: ${{ github.sha }}
+    secrets:
+      SECRETS_WORKLOAD_IDENTITY_PROVIDER: ${{ secrets.GCP_EHP_WORKLOAD_IDENTITY_PROVIDER }}
+      SECRETS_SERVICE_ACCOUNT: ${{ secrets.GCP_EHP_SERVICE_ACCOUNT }}
+
+  deploy-public-site-production:
+    needs:
+      - deploy-cms-production
+    uses: ./.github/workflows/reusable_public_site_deploy.yml
+    with:
+      environment: production
+      workload_identity_provider_secret_name: passculture-metier-ehp/gcp_metier_prod_workload_identity_provider 
+      cms_url: https://siteinstit-cms.production.passculture.team
+      backend_url: https://backend.passculture.team/
```

```suggestion
      backend_url: https://backend.passculture.team
```

pour être iso avec la value précédente

---

Il faut que je repercute ce changement dans la function de récuperation des données du backend côté next :)

---

Normalement c'est bon ! @lgerard-pass